### DISCUSSION TITLE
create-turbo with bunx

### DISCUSSION THREAD

**Discussion by DarkGuy10** [vercel/turborepo#9647] [Help]
### Summary

I was going through the quickstarter, and instead of npx I used bunx for running create-turbo.
I get this response:
```
$~ bunx -- bun create-turbo@latest
? Where would you like to create your Turborepo? ./whatever
? Which package manager do you want to use? Bun (beta)
>>> no "package.json" found at X:\GitHub\whatever
````
Has this always been like this? I've seen people mention creating turborepo like this.

### Additional information

```js
bun --version
1.1.42
```

**Comment by anthonyshew** [comment]
I'm unable to reproduce this. 🤔 
![CleanShot 2025-01-01 at 01 14 52@2x](https://github.com/user-attachments/assets/3963252a-a730-4307-a364-fad77d10d1ec)

**Comment by DarkGuy10** [comment]
![image](https://github.com/user-attachments/assets/e5798a98-4f75-465c-a2b4-67c3147b0f57)
Im onWindows 11 btw, does that make a difference?

**Reply by anthonyshew** [reply]
I would hope that it wouldn't but possibly. You may want to open an issue with Bun for this.

**Comment by tknickman** [comment]
@DarkGuy10 out of curiosity, can you try:
`bun create turbo`
